public/src/javascripts/caselist-date-carousel.js

```diff
@@ -0,0 +1,35 @@
+import Splide from '@splidejs/splide'
+
+if (document.querySelector('section.splide')) {
```

Comment:
Race condition, stick it inside

document.addEventListener('DOMContentLoaded', () => { ...

Reply:
yeah fair enough, I did originally have this in but thought it may not be needed as the external.js script is loaded at the very end? But just for completeness I'll put it back in